src/main/java/org/zone/region/group/key/GroupKeys.java

```diff
@@ -14,24 +14,26 @@
     BLOCK_PLACE("block_place", "Block Place"),
     INTERACT_DOOR("interact_door", "Interact with Door"),
     PLAYER_PREVENTION("prevent_players", "Prevent players from entering your zone"),
-    INTERACT_ITEMFRAME("interact_itemframe",
-                       "Prevent players from interacting an itemframe in your zone");
+    INTERACT_ITEMFRAME("interact_itemframe", "Prevent players from interacting an itemframe in " +
+            "your zone"),
+    ENTITY_DAMAGE_PLAYER("entity_damage_player", "Prevent damage from any other entity"),
+    PLAYER_FALL_DAMAGE("player_fall_damage", "Prevent player  fall damage");
 
-    private final @NotNull String name;
-    private final @NotNull String key;
+    private final String name;
+    private final String key;
 
     GroupKeys(@NotNull String key, @NotNull String name) {
         this.name = name;
         this.key = key;
     }
 
     @Override
-    public @NotNull String getName() {
+    public String getName() {
         return this.name;
     }
 
     @Override
-    public @NotNull String getKey() {
+    public String getKey() {
```

Comment:
Please add @NotNull back. Intellij should complain at you for not having it

Reply:
oh yes